src/hw1/bash_commands.py

```diff
@@ -0,0 +1,35 @@
+def head(file, strings=10):
+    for i in range(strings):
+        print(file.readline(), end="")
+
+
+def tail(file, strings=10):
+    lines = [line for line in file]
+    lines.reverse()
+    for i in range(strings):
+        print(lines[i], end="")
```

Comment:
Won't this print them backwards? Like, _in reverse order_, instead of just last lines?